includes/elements/container.php

```diff
@@ -428,9 +411,6 @@ protected function register_container_layout_controls() {
 						'placeholder' => $this->active_kit->get_settings_for_display( 'space_between_widgets' ),
 					],
 				],
-				'condition' => [
```

Comment:
Why this?

Reply:
This condition was link to the original hidden display: flex; control.